### PR TITLE
Update Fixed Point library to meet SRC-2 inline docs standard

### PR DESCRIPTION
**Pull request by bitzoic**
## Type of change

- Improvement (refactoring, restructuring repository, cleaning tech debt, ...)
- Documentation

## Changes

The following changes have been made:

- Update signed fixed point numbers to meet SRC-2 inline docs standard
- Update unsigned fixed point numbers to meet SRC-2 inline docs standard

### CODE REVIEW

File: libs/fixed_point/src/ifp128.sw [hunk 1]
```diff
@@ -24,27 +29,106 @@ impl From<UFP64> for IFP128 {
...
+    /// # Examples
+    ///
+    /// ```sway
+    /// use fixed_point::IFP128;
+    ///
+    /// fn foo() {
+    ///     let ifp128 = IFP128::zero();
+    ///     assert(ifp128.non_negative == true);
+    ///     let reverse = ifp128.sign_inverse();
+    ///     assert(reverse.non_negative == false);
```
Comment (Braqzen) [inline libs/fixed_point/src/ifp128.sw:129]:
```suggestion
    ///     assert(ifp128.non_negative);
    ///     let reverse = ifp128.sign_inverse();
    ///     assert(!reverse.non_negative);
```

File: libs/fixed_point/src/ifp128.sw [hunk 2]
```diff
@@ -237,7 +427,23 @@ impl IFP128 {
 }
 
 impl IFP128 {
-    /// Returns the nearest integer to `self`. Round half-way cases away from zero
+    /// Returns the nearest integer to `self`. Round half-way cases away from zero.
```
Comment (Braqzen) [inline libs/fixed_point/src/ifp128.sw:430]: > Round half-way cases away from zero.

?

File: libs/fixed_point/src/ifp256.sw [hunk 3]
```diff
@@ -24,27 +29,106 @@ impl From<UFP128> for IFP256 {
...
+    /// # Examples
+    ///
+    /// ```sway
+    /// use fixed_point::IFP256;
+    ///
+    /// fn foo() {
+    ///     let ifp256 = IFP256::zero();
+    ///     assert(ifp256.non_negative == true);
+    ///     let reverse = ifp256.sign_inverse();
+    ///     assert(reverse.non_negative == false);
```
Comment (Braqzen) [inline libs/fixed_point/src/ifp256.sw:129]:
```suggestion
    ///     assert(ifp256.non_negative);
    ///     let reverse = ifp256.sign_inverse();
    ///     assert(!reverse.non_negative);
```

File: libs/fixed_point/src/ifp64.sw [hunk 4]
```diff
@@ -24,27 +31,106 @@ impl From<UFP32> for IFP64 {
...
+    /// # Examples
+    ///
+    /// ```sway
+    /// use fixed_point::IFP64;
+    ///
+    /// fn foo() {
+    ///     let ifp64 = IFP64::zero();
+    ///     assert(ifp64.non_negative == true);
+    ///     let reverse = ifp64.sign_inverse();
+    ///     assert(reverse.non_negative == false);
```
Comment (Braqzen) [inline libs/fixed_point/src/ifp64.sw:131]:
```suggestion
    ///     assert(ifp64.non_negative);
    ///     let reverse = ifp64.sign_inverse();
    ///     assert(!reverse.non_negative);
```

**Review by Braqzen — COMMENTED** [review]
Fields on some of the struct are undocumented

File: libs/fixed_point/src/ifp128.sw (hunk 2 above)
Comment (bitzoic) [reply libs/fixed_point/src/ifp128.sw:432]: 0.5 -> 0
0.51 -> 1

**Review by Braqzen — APPROVED** [review]

**Pull request merged**